### PR TITLE
H2IP-1137 Added related search component

### PR DESCRIPTION
**Pull request by abdullahwaqas-ae**

### CODE REVIEW

File: src/components/ui/RelatedSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useHawksearch } from 'components/StoreProvider';
+
+function RelatedSearch() {
+	const {
+		actor: hawkActor,
+		store: { searchResults },
+	} = useHawksearch();
+	const relatedFacet = searchResults && searchResults.Facets.find(facet => facet.FacetType === 'related');
+	console.log(relatedFacet);
```
Comment (lukasz-bargiel-hs) [inline src/components/ui/RelatedSearch.tsx]: please remove this ;)

File: src/components/ui/RelatedSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+			Keyword: keyword,
+		});
+	}
+	return (
+		<>
+			{relatedFacet && (
+				<div className="hawk-related_search-container">
+					<span className="heading">Related Searches: </span>
+					{relatedFacet.Values.map((item, index) => {
+						console.log(index, relatedFacet.Values.length);
```
Comment (lukasz-bargiel-hs) [inline src/components/ui/RelatedSearch.tsx]: this also need to be removed

File: src/components/ui/RelatedSearch.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+						return (
+							<span
+								key={index}
+								onClick={() => searchWithKeyword(item.Value)}
+								className="related-searched-words"
+							>
+								{' '}
```
Comment (lukasz-bargiel-hs) [inline src/components/ui/RelatedSearch.tsx]: This part is hard to understand - is there a chance to build it in a way that would be understandable without deeper look /

File: src/components/ui/RelatedSearch.tsx (hunk 1 above)
Comment (abdullahwaqas-ae) [reply src/components/ui/RelatedSearch.tsx]: Removed

File: src/components/ui/RelatedSearch.tsx (hunk 2 above)
Comment (abdullahwaqas-ae) [reply src/components/ui/RelatedSearch.tsx]: Removed

File: src/components/ui/RelatedSearch.tsx (hunk 3 above)
Comment (abdullahwaqas-ae) [reply src/components/ui/RelatedSearch.tsx]: Simplified the logic

**Pull request merged**